src/command-line/start.js

```diff
@@ -13,22 +13,25 @@ program
 	.description("Start the server")
 	.action(function() {
 		var users = new ClientManager().getUsers();
-		var config = Helper.getConfig();
-		var mode = config.public;
+
+		var mode = Helper.config.public;
 		if (program.public) {
 			mode = true;
 		} else if (program.private) {
 			mode = false;
 		}
+
 		if (!mode && !users.length) {
 			log.warn("No users found!");
 			log.info("Create a new user with 'lounge add <name>'.");
-		} else {
-			server({
-				host: program.host || process.env.IP   || config.host,
-				port: program.port || process.env.PORT || config.port,
-				bind: program.bind || config.bind,
-				public: mode
-			});
+
+			return;
 		}
+
+		Helper.config.host = program.host || process.env.IP   || Helper.config.host;
```

Comment:
I don't really see a use for it. I would get rid of it. I think the only cases we should use environment variables is for things that can't be in the config file (so, the config path).